packages/keeper-bots/src/providers/pancake-swap.ts

```diff
@@ -18,6 +18,10 @@ class PancakeSwapProvider extends SwapProvider {
   private v3PancakeSubgraphClient: UrqlClient | undefined;
   private tokens: Map<Address, Token>;
   private quoteProvider: QuoteProvider;
+  private _candidatePools: {
+    timestamp: Date;
```

Comment:
First I thought this was when the candidate pools were last fetched, but this is more of the timestamp in the future when they will be invalid, maybe a better name is `expirationTimestamp`?